server/memphis_sdk_handlers.go

```diff
@@ -167,6 +167,16 @@ func (s *Server) initializeSDKHandlers() {
 		"memphis_schema_detachments_listeners_group",
 		detachSchemaHandler(s))
 
+	//schema creation
+	s.queueSubscribe(globalAccountName, "$memphis_schema_creations",
```

Comment:
you can move it to be under schemas section together with memphis_schema_attachments